src/ds/ds-devices-common.cpp

```diff
@@ -0,0 +1,329 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
```

Comment:
As discussed rename file to `ds-device-common`

Reply:
Done